Classes/Image Upload/ImageUploadTableViewController.swift

```diff
@@ -186,13 +196,35 @@ class ImageUploadTableViewController: UITableViewController, UITextFieldDelegate
             }
         }
     }
+    
+}
 
-    // MARK: UITextFieldDelegate
+// MARK: UITextFieldDelegate
 
+extension ImageUploadTableViewController: UITextFieldDelegate {
     /// Called when the user taps return on the title field, moves their cursor to the body
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
         bodyTextField.becomeFirstResponder()
         return false
     }
+}
 
+// MARK: UITextViewDelegate
+
+extension ImageUploadTableViewController: UITextViewDelegate {
+    func textViewDidBeginEditing(_ textView: UITextView) {
+        if textView.text == bodyPlaceholder {
```

Comment:
Maybe we should just start w/ `Optional` in the text field and add this later if totally necessary. Doing this text comparison is pretty inefficient. `SLKTextView` has an implementation for placeholders w/ `UITextView` already too.

Reply:
I didn't know about `SLKTextView`. I'll take a look and try to do with it, ok?

Reply:
See here: https://github.com/slackhq/SlackTextViewController/blob/master/Source/SLKTextView.h